imageviewer.js

```diff
@@ -740,49 +770,47 @@
         snapView: true,
         maxZoom: 500,
         refreshOnResize: true,
-        zoomOnMouseWheel : true
+        zoomOnMouseWheel: true
     }
 
-    window.ImageViewer = function (container, options) {
-        var imgElm, imgSrc, hiResImg;
-        if (!(container && (typeof container == "string" || container instanceof Element || container[0] instanceof Element))) {
-            options = container;
-            container = $('#iv-container');
+    window.ImageViewer = function(container, options) {
+        var imgElm, imgSrc, hiResImg
+        if (!(
+                container &&
+                (typeof container == 'string' || container instanceof Element || container[0] instanceof Element)
+            )) {
+            options = container
+            container = document.getElementById('iv-container')
```

Comment:
Let's use querySelector and querySelectorAll for all dom traversal. This are  little slow than getElementById / getElementByClassName , but are more abstracted.